UnicornRender/video/source/Texture.cpp

```diff
@@ -42,12 +47,12 @@ bool Texture::Load(const std::string& path)
 
     if (!textureHandler.IsValid())
     {
-        LOG_ERROR("Can't find texture - %s", m_path.c_str());
+        spdlog::get(utility::Logger::LoggerName())->error("Can't find texture - %s", m_path.c_str());
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.